wagtail/admin/wagtail_hooks.py

```diff
@@ -247,3 +257,165 @@ def register_core_features(features):
+            'type': BLOCK_TYPES.UNORDERED_LIST_ITEM, 'icon': 'list-ul'
+        })
+    )
+    features.register_converter_rule('contentstate', 'ul', {
+        'from_database_format': {
+            'ul': ListElementHandler('unordered-list-item'),
+            'li': ListItemElementHandler(),
+        },
+        'to_database_format': {
+            'block_map': {BLOCK_TYPES.UNORDERED_LIST_ITEM: {'element': 'li', 'wrapper': 'ul'}}
+        }
+    })
+    features.register_editor_plugin(
+        'draftail', 'ol', draftail_features.BlockFeature({
+            'type': BLOCK_TYPES.ORDERED_LIST_ITEM, 'icon': 'list-ol'
+        })
+    )
+    features.register_converter_rule('contentstate', 'ol', {
+        'from_database_format': {
+            'ol': ListElementHandler('ordered-list-item'),
+            'li': ListItemElementHandler(),
+        },
+        'to_database_format': {
+            'block_map': {BLOCK_TYPES.ORDERED_LIST_ITEM: {'element': 'li', 'wrapper': 'ol'}}
+        }
+    })
+
+    features.register_editor_plugin(
+        'draftail', 'bold', draftail_features.InlineStyleFeature({
+            'type': INLINE_STYLES.BOLD, 'icon': 'bold'
+        })
+    )
+    features.register_converter_rule('contentstate', 'bold', {
+        'from_database_format': {
+            'b': InlineStyleElementHandler('BOLD'),
+            'strong': InlineStyleElementHandler('BOLD'),
+        },
+        'to_database_format': {
+            'style_map': {INLINE_STYLES.BOLD: 'b'}
+        }
+    })
+    features.register_editor_plugin(
+        'draftail', 'italic', draftail_features.InlineStyleFeature({
+            'type': INLINE_STYLES.ITALIC, 'icon': 'italic'
+        })
+    )
+    features.register_converter_rule('contentstate', 'italic', {
+        'from_database_format': {
+            'i': InlineStyleElementHandler('ITALIC'),
+            'em': InlineStyleElementHandler('ITALIC'),
+        },
+        'to_database_format': {
+            'style_map': {INLINE_STYLES.ITALIC: 'i'}
+        }
+    })
+
+    features.register_editor_plugin(
+        'draftail', 'link', draftail_features.EntityFeature({
+            'type': ENTITY_TYPES.LINK,
+            'icon': 'link',
+            'source': 'LinkSource',
+            'decorator': 'Link',
+            # We want to enforce constraints on which links can be pasted into rich text.
+            # Keep only the attributes Wagtail needs.
+            'attributes': ['url', 'id', 'parentId'],
+            # Keep only links which are not anchor references.
+            'whitelist': {
+                'href': '^(?!#)',
+            }
+        })
+    )
+    features.register_converter_rule('contentstate', 'link', {
+        'from_database_format': {
+            'a[href]': ExternalLinkElementHandler('LINK'),
+            'a[linktype="page"]': PageLinkElementHandler('LINK'),
+        },
+        'to_database_format': {
+            'entity_decorators': {ENTITY_TYPES.LINK: link_entity}
+        }
```

Comment:
All of the `BlockElementHandler`, `ListElementHandler `, `InlineStyleElementHandler`, `ExternalLinkElementHandler` use the Draft.js type strings – is there any reason not to use the `ENTITY_TYPES`, `BLOCK_TYPES`, `INLINE_STYLES` constants for this too? Or on the contrary, why use the constants elsewhere then? I think this should be consistent either way.